ProgressBar/themes/bootstrap/variables.less

```diff
@@ -1,2 +1,3 @@
-@transition-params: 0.3s linear 0s;
-@line-height: 1.6em;
\ No newline at end of file
+@height: 1.8em; // Controls the height of the widget (180% of the font-size)
```

Comment:
isn't that "dangerous" to have such non-scoped LESS (i.e. not pb-height for example) variable naming just for a particular widget? What is our practice elsewhere? Can this collapse with other widget defining a similar variable?